src/api/routes/interviews.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * API for interviews.
+ * An interview is always bound to a user.
+ * Therefore, every endpoint requires an authentication.
+ */
+
+module.exports = (db, security) => {
+    const router = require('express').Router()
+    const validate = security.validate()
+    const messageKeys = require('../util/message-keys')
+
+    /**
+     * Get all interviews of a user.
+     * If now interviews are found, an empty array is being returned.
+     */
+    async function getInterviews(req, res) {
+        const userId = req.user.data.id
+        try {
+            const interviews = await db.getInterviews(userId)
+            if (interviews === null || !Array.isArray(interviews)) {
```

Comment:
Consider responding with status code 500 if the `interviews` variable is not an array. In which _normal_ case would that happen anyways?

Reply:
The lowdb implementation does not say anything but the db should respond with `null` or an array. Therefore, 500 seems reasonable.

Reply:
Update: It is not the controller's duty to validate the db interface return values. No check should be implemented here.